packages/sources/synthetix-debt-pool/src/utils.ts

```diff
@@ -67,3 +77,41 @@ export const getContractAddress = async (
   const contractNameBytes32 = ethers.utils.formatBytes32String(contractName)
   return await addressResolver.getAddress(contractNameBytes32)
 }
+
+export const getSynthetixBridgeName = (networkName: string, jobRunID: string): string => {
+  if (networkName === SupportedChains.ETHEREUM || networkName === SupportedChains.KOVAN)
```

Comment:
Just a sanity check. Why would we return 'SynthetixBridgeToOptimism' if neworkName === SupportedChains.ETHEREUM and vis versa? Are these switched

Reply:
Keyword is _to_ Optimism. So from Ethereum to Optimism.